src/test/java/seedu/realodex/logic/parser/ParserUtilTest.java

```diff
@@ -440,4 +444,19 @@ public void parseRemark_validValueWithWhitespace_returnsTrimmedRemark() throws E
         Remark expectedRemark = new Remark(VALID_REMARK_ONE);
         assertEquals(expectedRemark, ParserUtil.parseRemark(remarkWithWhitespace));
     }
+
+    @Test
```

Comment:
Thanks for adding the tests!

Reply:
code coverage is still not hitting targets, i can pick it up from here after merging

Reply:
Can alr i added some tests

Reply:
scary